VideoLocker/src/main/java/org/edx/mobile/view/MyCourseListFragment.java

```diff
@@ -0,0 +1,257 @@
+import org.edx.mobile.interfaces.NetworkObserver;
+import org.edx.mobile.interfaces.NetworkSubject;
+import org.edx.mobile.loader.AsyncTaskResult;
+import org.edx.mobile.loader.CoursesAsyncLoader;
+import org.edx.mobile.logger.Logger;
+import org.edx.mobile.model.api.EnrolledCoursesResponse;
+import org.edx.mobile.module.analytics.ISegment;
+import org.edx.mobile.module.prefs.LoginPrefs;
+import org.edx.mobile.util.NetworkUtil;
+import org.edx.mobile.util.ViewAnimationUtil;
+import org.edx.mobile.view.adapters.MyCourseAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import de.greenrobot.event.EventBus;
+
+public class MyCourseListFragment extends BaseFragment implements NetworkObserver, LoaderManager.LoaderCallbacks<AsyncTaskResult<List<EnrolledCoursesResponse>>> {
+
+    private static final int MY_COURSE_LOADER_ID = 0x905000;
+
+    private MyCourseAdapter adapter;
+    private FragmentMyCourseListBinding binding;
+    private Logger logger = new Logger(getClass().getSimpleName());
+    private boolean refreshOnResume = false;
+
+    @Inject
+    IEdxEnvironment environment;
+
+    @Inject
+    LoginPrefs loginPrefs;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        adapter = new MyCourseAdapter(getActivity(), environment) {
+            @Override
+            public void onItemClicked(EnrolledCoursesResponse model) {
+                MyCourseListFragment.this.environment.getRouter().showCourseDashboardTabs(getActivity(), MyCourseListFragment.this.environment.getConfig(), model, false);
+            }
+
+            @Override
+            public void onAnnouncementClicked(EnrolledCoursesResponse model) {
+                environment.getRouter().showCourseDashboardTabs(getActivity(), environment.getConfig(), model, true);
+            }
+        };
+        environment.getSegment().trackScreenView(ISegment.Screens.MY_COURSES);
+        EventBus.getDefault().register(this);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        binding = DataBindingUtil.inflate(inflater, R.layout.fragment_my_course_list, container, false);
+        binding.swipeContainer.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                // Hide the progress bar as swipe layout has its own progress indicator
+                binding.loadingIndicator.getRoot().setVisibility(View.GONE);
+                binding.noCourseTv.setVisibility(View.GONE);
+                loadData(false);
+            }
+        });
+        binding.swipeContainer.setColorSchemeResources(android.R.color.holo_blue_bright,
+                R.color.grey_act_background, R.color.grey_act_background,
+                R.color.grey_act_background);
```

Comment:
And instead of these we can use `edx_grayscale_neutral_xx_light` or `edx_grayscale_neutral_x_light`.